homeassistant/components/onvif/event.py

```diff
@@ -76,11 +76,9 @@ async def async_start(self) -> bool:
         if await self.device.create_pullpoint_subscription():
             # Initialize events
             pullpoint = self.device.create_pullpoint_service()
-            await pullpoint.SetSynchronizationPoint()
```

Comment:
Instead of removing this, can we put it in a `try/except` block and catch the exception thrown for your camera? This call is explicitly needed in order to get initial event statuses from some of the camera models I own.

Reply:
Done!